optimade/server/routers/utils.py

```diff
@@ -255,7 +259,9 @@ def get_entries(
         # Deduce the `next` link from the current request
         query = urllib.parse.parse_qs(request.url.query)
         query["page_offset"] = int(query.get("page_offset", [0])[0]) + len(results)
-        urlencoded = urllib.parse.urlencode(query, doseq=True)
+        urlencoded = urllib.parse.urlencode(
+            query, doseq=True, quote_via=dummy_quote
+        )  # it seems the url is also quoted at a later stage so we should not quote it here, hence the dummy_quote.
```

Comment:
```suggestion
        urlencoded = urllib.parse.urlencode(
            query, doseq=True, quote_via=lambda x, *_: x.replace(" ", "+")
        )
```

is probably a bit cleaner. I had a look at the built-in `urllib.parse.quote` method, not sure how it differs from this when passed to `quote_via`.

Reply:
Your suggestion is indeed cleaner. I'll use a lambda function instead.